skabase/SKABaseDevice/SKABaseDevice.xmi

```diff
@@ -34,6 +34,22 @@
       <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
       <DefaultPropValue>localhost</DefaultPropValue>
     </deviceProperties>
+    <deviceProperties name="CentralLogger" description="">
+      <type xsi:type="pogoDsl:StringType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+    </deviceProperties>
+    <deviceProperties name="ElementLogger" description="">
+      <type xsi:type="pogoDsl:StringType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+    </deviceProperties>
+    <deviceProperties name="ElementLoglevel" description="">
+      <type xsi:type="pogoDsl:UShortType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+    </deviceProperties>
+    <deviceProperties name="ServerHostName" description="">
```

Comment:
The device is unlikely to decide which host it will run on, so why specify this as a property?  Wouldn't it be better for the device to report back where it happens to be running using a read-only attribute?

Reply:
Removing it.